src/Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -2723,7 +2723,7 @@ public static GeneratorRunResult VerifyOutputsMatch(this GeneratorRunResult actu
 
         private static string TrimChecksum(string text)
         {
-            var trimmed = text.Trim('\r', '\n').Replace("\r\n", "\n").Replace('\n', '\r');
+            var trimmed = text.Trim('\r', '\n').Replace("\r\n", "\r").Replace('\r', '\n');
```

Comment:
don't we have a .NormalizeWhitespace() or similar helper that can remove the guesswork here?

Reply:
We're in the SDK repo, so no, this is all new custom test stuff. We're going to move this over to the razor compiler repo, and I imagine i'll pull in some of the roslyn testing bits when we do.